snyk/managers.py

```diff
@@ -366,6 +369,7 @@ def all(self) -> Any:
         raise SnykError
 
 
+@deprecated("API has been removed, use IssueSetAggregatedManager instead")
```

Comment:
@jgresty following this change what would be the behavior of a call to an IssueSetManager method?

Reply:
It throws a `DeprecationWarning` when you create an instance of the class but any method calls are silent